css/scss/sections/_index.scss

```diff
@@ -0,0 +1,228 @@
+body {
+  background-color: #307367;
+  @include breakpoint(medium) {
+  background-color: #fff;
+  }
+}
+
+
+p{
```

Comment:
Same here.
You're changing the entire project p tags to this styling.

Reply:
Same as above. You need to create a custom class and style that class, or style the nested p tag, inside of one specific class. The way you wrote the p selector is not nested, meaning that it will affect all p tags on the entire site.
Add a custom/specific class for the paragraph you want to style.